orttraining/orttraining/training_api/module.cc

```diff
@@ -278,47 +283,55 @@ Module::Module(const std::string& train_model_path_or_bytes,
     }
   }
 
-  if (eval_model_path_or_bytes.has_value()) {
+  if (model_identifiers.eval_model.has_value()) {
+    eval_sess_ = std::make_unique<onnxruntime::InferenceSession>(session_options, env);
+    ORT_THROW_IF_ERROR(eval_sess_->Load(model_identifiers.eval_model.value()));
+  } else if (model_identifiers.eval_model_data.size() != 0) {
```

Comment:
I guess there will be a compiler error on win without this change
```suggestion
  } else if (model_identifiers.eval_model_data.size() != 0L) {
```